build/tpl.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2013-2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ *
+ */
+/**
+ * ORGINAL VERSION:
+ * https://github.com/epeli/requirejs-hbs
+ * Copyright 2013 Esa-Matti Suuronen
+ * MIT License : https://github.com/epeli/requirejs-hbs/blob/master/LICENSE
+ *
+ * MODIFIED VERSION:
+ * @author Bertrand Chevrier <bertrand@taotesting.com> for OAT SA
+ * - Minor code refactoring
+ * - i18n helper has been added
+ * - dompurify helper has been added
+ */
+
+import __ from 'i18n';
+import _ from 'lodash';
+import DOMPurify from 'lib/dompurify/purify';
+
+export default hb => {
```

Comment:
those helpers will be needed on most of the pacakges and on tao.
I think they should be part of `tao-core-libs`

Reply:
Is it part of this task, or should we create a new task about extract `tao-core-ui` lib dependencies into `tao-core-libs`?

Reply:
let's do it after then